llarp/util/time.cpp

```diff
@@ -74,4 +78,27 @@ namespace llarp
     out.fill(old_fill);
     return out << "s";
   }
+
+  std::ostream&
+  operator<<(std::ostream& out, const TimePoint_t& tp)
+  {
+    auto t = TimePoint_t::clock::to_time_t(tp);
+    return out << std::put_time(std::localtime(&t), "%c %Z");
+  }
+  namespace util
+  {
+    std::string
+    time_delta_string(const TimePoint_t& at)
+    {
+      std::stringstream ss;
+      const auto dlt = TimePoint_t::clock::now() - at;
+      if (dlt > 0s)
+        ss << std::chrono::duration_cast<Duration_t>(TimePoint_t::clock::now() - at) << " ago ";
+      else if (dlt < 0s)
+        ss << "in " << std::chrono::duration_cast<Duration_t>(TimePoint_t::clock::now() - at);
```

Comment:
I think this is supposed to be negated, i.e. `at - now`, or alternatively:
```suggestion
        ss << "in " << std::chrono::duration_cast<Duration_t>(-dlt);
```